requirements.txt

```diff
@@ -4,4 +4,5 @@ keras==2.1.1
 tensorflow==1.3.0
 h5py==2.7.1
 opencv-python==3.4.0.12
-scikit-learn==0.19.1 
\ No newline at end of file
+scikit-learn==0.19.1 
+lxml==4.2.3
```

Comment:
Should I change 
`lxml==4.2.3` to `lxml!=4.2.2`

Reply:
No, pinning to 4.2.3 is better, this way we're sure all users get the same version we are using.